.ansible-lint

```diff
@@ -1,11 +1,3 @@
-exclude_paths:
-  - .
-  - temp
-parseable: true
-quiet: true
 skip_list:
-  - "301"
-  - "305"
-  - "306"
-use_default_rules: true
-verbosity: 1
+  - "schema[playbook]"
```

Comment:
why are we skipping the playbook schema linter, this seems like it would be useful?

Reply:
It had a problem with the `role:` in the ansible playbook.  if the role is removed then the schema check passes although the role works fine.     
```
schema[playbook]: {'name': 'Main', 'hosts': 'all', 'become': True, 'roles': [{'name': 'geerlingguy.docker', 'vars': {'docker_edition': 'ce', 'docker_packages': ['docker-{{ docker_edition }}', 'docker-{{ docker_edition }}-cli', 'docker-{{ docker_edition }}-rootless-extras'], 'docker_packages_state': 'present', 'docker_users': ["{{ lookup('env','SSH_USERNAME') }}"]}}, {'name': 'trallnag.awscli', 'vars': {'awscli_version': '2.11.21', 'awscli_ssm_version': '1.2.463.0'}}], 'tasks': [{'name': 'Install packages', 'ansible.builtin.apt': {'state': 'present', 'update_cache': True, 'pkg': ['curl', 'jq', 'ntpdate', 'python3-pip', 'unzip']}}, {'name': 'Update time', 'ansible.builtin.command': 'ntpdate -u pool.ntp.org', 'changed_when': False}, {'name': 'Make aws apps directory', 'ansible.builtin.file': {'path': '/opt/aws/bin', 'state': 'directory', 'mode': '0755'}}, {'name': 'Download CFN helper scripts', 'ansible.builtin.get_url': {'url': 'https://s3.amazonaws.com/cloudformation-examples/aws-cfn-bootstrap-py3-latest.tar.gz', 'dest': '/tmp/aws-cfn-bootstrap-py3-latest.tar.gz', 'force': True}}, {'name': 'Install CFN helper scripts', 'ansible.builtin.pip': {'name': 'file:///tmp/aws-cfn-bootstrap-py3-latest.tar.gz', 'extra_args': '--target=/opt/aws/bin'}}]} is not valid under any of the given schemas
src/playbook.yaml:1  Returned errors will not include exact line numbers, but they will mention
the schema name being used as a tag, like ``schema[playbook]``,
``schema[tasks]``
```

I'm not sure why the error is happening but I'm guessing that the schema used for validating is just lagging an update.  In any case, i managed to figure out how to replace the `role:` with `ansible.builtin.import_role:` in PR #3 .  I'll remove this from the skip list but that means validation won't pass until #3 is merged.